.circleci/config.yml

```diff
@@ -756,6 +766,19 @@ jobs:
       - react-get-deps
       - run: make -C webui/react check
 
+  build-react-preview:
+    docker:
+      - image: cimg/node:12.16
+    environment:
+      PUBLIC_URL: /0
+    steps:
+      - checkout
+      - react-get-deps
+      - run: make -C webui/react build
+      - store_artifacts:
+          path: webui/react/build
+          destination: /
+
```

Comment:
**suggestion:** I think all the CI config changes look good from what I can tell, but might be good to have someone more knowledgable look at the CircleCI changes. Maybe @rb-determined-ai ?

Reply:
Not sure I understand quite what the intention is... why is this an artifact?  Is it to save build times somewhere else or is it something that is generally useful after the build?

Reply:
It's more of the latter. The idea is to hang onto the react build files when a PR fires off a CI job. When it completes, anybody can go from the PR to the CI job and look at the artifacts to view the WebUI and try out the changes made without having to build the branch locally.

Reply:
Can you meaningfully use the build artifact without a live master running?  That's the part I think I'm missing

Reply:
I can't remember which one the CI will point to but it will point to a live master. I believe it's `latest-master`...

Reply:
oh neat.  Yeah this sounds cool

Reply:
100% @hamidzr's idea

Reply:
the accompanying change for latest-master https://github.com/determined-ai/determined/pull/1439